package Nested.Exception;

public class TestThrow1{  
	   static void validate(int age){  
	     if(age<18)  
	      throw new ArithmeticException("not valid");  
	     else  
	      System.out.println("welcome to vote");  
	   }  
	   public static void main(String args[]){  
	      validate(13);  
	      System.out.println("rest of the code...");  
	  }  
	}  

/*Java throw keyword
The Java throw keyword is used to explicitly throw an exception*/